src/Uno.Templates/content/unoapp/MyExtensionsApp._1/App.recommended.cs

```diff
@@ -123,7 +125,7 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
                         credentials ??= new Dictionary<string, string>();
                         credentials[TokenCacheExtensions.AccessTokenKey] = "SampleToken";
                         credentials[TokenCacheExtensions.RefreshTokenKey] = "RefreshToken";
-                        credentials["Expiry"] = DateTime.Now.AddMinutes(5).ToString("g");
+                        credentials["Expiry"] = DateTime.Now.AddMinutes(5).ToString("g", DateTimeFormatInfo.InvariantInfo);
```

Comment:
NOTE: This is a behavioral change. When using a null provider, the used default will be CurrentInfo which doesn't seem suitable here.

https://github.com/dotnet/runtime/blob/4001e074ee648506d64277729a7cce089a4996e2/src/libraries/System.Private.CoreLib/src/System/Globalization/DateTimeFormatInfo.cs#L314